Docs/source/install/hpc/summit.rst

```diff
@@ -264,6 +265,9 @@ Known System Issues
    The above **HDF5 Jupyter read** work-around for OLCFHELP-3685 does not work anymore, due to the way that GPFS is mounted via NSF on Jupyter nodes.
    As a work-around until this is fixed, please copy your HDF5 data to ``/ccs``, ``$HOME`` or use ADIOS2 BP instead of HDF5 files.
 
+   Jan 18th, 2023:
+   The work-around above works again with GPFS data.
+
```

Comment:
Should we remove this whole warning then? 
We can still see the history of this from the `git` history, if we need it.

Reply:
True, simpler. And we have also versions in our RTD pages.